File: ManagerApplication/ManagerApp/src/app/model/building.ts
import { Floor } from "./floor";

export class Building {
    id: number;
    x: number;
    y: number;
    width: number;
    height: number;
    name: string;
    type: TypeOfBuilding;

    constructor(id: number,x: number,y: number,width: number,height: number,name:string,type:TypeOfBuilding){
        this.id = id;
        this.x = x;
        this.y = y;
        this.width = width;
        this.height = height;
        this.name = name;
        this.type = type;
    }

    
}

export enum TypeOfBuilding{
    Hospital,
    Parking
}

File: HealthCareClinic/Hospital/Rooms and equipment/Model/Building.cs

using System;
using System.Collections.Generic;
using System.Text;

namespace Hospital.Rooms_and_equipment.Model
{
    public class Building
    {
        public int Id
        {
            get; set;
        }
        public enum BuildingType
        {
            Hospital,
            Parking
        }

        public String Name
        {
            get; set;
        }

        public BuildingType Type
        {
            get; set;
        }

        public float X
        {
            get; set;
        }
        public float Y
        {
            get; set;
        }
        public float Width
        {
            get; set;
        }
        public float Height
        {
            get; set;
        }

        public Building(int id, string name, BuildingType type, float x, float y, float width, float height)
        {
            Id = id;
            Name = name;
            Type = type;
            X = x;
            Y = y;
            Width = width;
            Height = height;
        }

        public Building() {
        }
    }
}
